test/fakeOrigin/docs/Configuration.md

```diff
@@ -0,0 +1,122 @@
+# Configuration
```

Comment:
So I guess this is one of those Markdown files you said couldn't reliably contain comments - but you're using GitHub extensions so it seems like you're expecting people to view this on GitHub so... I guess my question is just: why not?

Reply:
Addressed with new commits